openapi.yaml

```diff
@@ -4220,56 +4204,7 @@ paths:
         - $ref: '#/components/parameters/sortFields'
       responses:
         '200':
-          description: OK
```

Comment:
Deleted custom response object and updated it to be using a reference to 'Workspaces' model